src/libopensc/pkcs15-oberthur.c

```diff
@@ -734,6 +733,9 @@ sc_pkcs15emu_oberthur_add_cert(struct sc_pkcs15_card *p15card, unsigned int file
 
 	rv = sc_oberthur_get_certificate_authority(&cinfo.value, &cinfo.authority);
 	if (rv != SC_SUCCESS) {
+#ifdef ENABLE_OPENSSL
+		sc_log_openssl(ctx);
+#endif
```

Comment:
is the ifdef needed? I think the `sc_log_openssl()` is defined also when built without openssl as noop. This is used in few more places.